src/github/utils.ts

```diff
@@ -12,21 +12,89 @@ import { parseDiffHunk, DiffHunk } from '../common/diffHunk';
 import * as Common from '../common/timelineEvent';
 import * as GraphQL from './graphql';
 import { Resource } from '../common/resources';
+import { PullRequestModel } from './pullRequestModel';
+import { getEditCommand, getDeleteCommand, getCommentThreadCommands } from './commands';
+import { PRNode } from '../view/treeNodes/pullRequestNode';
+import { ReviewDocumentCommentProvider } from '../view/reviewDocumentCommentProvider';
+import { uniqBy } from '../common/utils';
+
+export interface CommentHandler {
+	commentController?: vscode.CommentController;
+	startReview(thread: vscode.CommentThread): Promise<void>;
+	finishReview(thread: vscode.CommentThread): Promise<void>;
+	deleteReview(): Promise<void>;
+	createOrReplyComment(thread: vscode.CommentThread): Promise<void>;
+	editComment(thread: vscode.CommentThread, comment: vscode.Comment): Promise<void>;
+	deleteComment(thread: vscode.CommentThread, comment: vscode.Comment): Promise<void>;
+}
 
-export function convertToVSCodeComment(comment: Comment, command?: vscode.Command): vscode.Comment {
-	return {
+export function convertToVSCodeComment(comment: Comment, command: vscode.Command | undefined): vscode.Comment & { _rawComment: Comment } {
+	let vscodeComment: vscode.Comment & { _rawComment: Comment } = {
+		_rawComment: comment,
 		commentId: comment.id.toString(),
 		body: new vscode.MarkdownString(comment.body),
-		command: command,
+		selectCommand: command,
 		userName: comment.user!.login,
-		gravatar: comment.user!.avatarUrl,
-		canEdit: comment.canEdit,
-		canDelete: comment.canDelete,
-		isDraft: !!comment.isDraft,
+		userIconPath: vscode.Uri.parse(comment.user!.avatarUrl),
+		label: !!comment.isDraft ? 'Pending' : undefined,

```

Comment:
This seems to be an upstream issue, if I start a review, and then reload vscode, the comments are not initially shown with the 'Pending' label. I see that the label is correctly being set here. If I add another pending comment to the thread, the labels for the thread are rendered